src/Assets/Theme.ts

```diff
@@ -12,10 +12,10 @@ export default {
       lg: 76, // rem
     },
     breakpoints: {
-      xs: 0, // em
-      sm: 48, // em
-      md: 64, // em
-      lg: 75, // em
+      xs: 768, // px
+      sm: 900, // px
+      md: 1024, // px
+      lg: 1192, // px
```

Comment:
We might want to import the breakpoints from palette. I also noticed we aren't using the exact same breakpoints as we are anywhere else. There might just be some historical reason, but is there any chance we could possibly switch over to palette's `<Theme>`?

```jsx
import { breakpoints } from "@artsy/palette"

export default {
  ...

  publishing: {
    breakpoints: {
      xs: breakpoints.sm,
      sm: breakpoints.md,
      md: breakpoints.lg,
      lg: breakpoints.xl,
    }
  },
  ...
}
```

Reply:
yes this on my list of todo's, noted in PR description.

Reply:
The main theme breakpoints are now updated, leaving the publishing ones for the moment as they are not yet using most of the palette stuff, will be a project on its own to see that everything is working as expected.